### PR TITLE
Google Pay payment method support for 3-D Secure

### PR DESCRIPTION
**Pull request by ct-clearhaus**
The title says it all.

### CODE REVIEW

**Comment by mt-clearhaus** [comment]
Current rendering:

Screenshot from 2023-12-12 15-29-09

Screenshot from 2023-12-12 15-29-36

**Comment by mt-clearhaus** [comment]
Current rendering:

Screenshot from 2023-12-12 15-29-09

Screenshot from 2023-12-12 15-33-47

**Comment by mt-clearhaus** [comment]
Current rendering:

Screenshot from 2023-12-13 13-07-58

Screenshot from 2023-12-13 13-08-30

**Review by ad-clearhaus — APPROVED** [review]
I approve of the text (and wording). :+1:

Google Pay has a lot of **Notice**s. :see_no_evil: And it makes sense, Google Pay has a lot of corner cases.

File: website/content/gateway/api_reference/resources/authorizations/authorizations_methods/googlepay.md [hunk 1]
```diff
@@ -24,14 +24,21 @@ The shared secret derived from the ephemeral public key and your private key
...
+{{% description_details %}}
+See [Authentication: [3dsecure]](#authentication-3dsecure).
+{{% regex_optional %}}Optional{{% /regex_optional %}}
+
 {{% /description_list %}}
 {{% notice %}}
 **Notice**: Signing is required to use the `googlepay` payment method.
 
-**Notice**: An authorization made with `googlepay` is strongly authenticated (SCA in PSD2) if `authMethod` is `CRYPTOGRAM_3DS` and the Google Pay guidelines for SCA (link) have been followed. If authMethod is PAN_ONLY, a 3-D Secure flow is required for SCA.
+**Notice**: An authorization made with `googlepay` is strongly authenticated (SCA in PSD2) if `authMethod` is `CRYPTOGRAM_3DS` and the [Google Pay guidelines for SCA](https://developers.google.com/pay/api/android/guides/resources/sca) have been followed. If `authMethod` is `PAN_ONLY`, a 3-D Secure flow is required for SCA and the resulting ARes/RReq must be supplied in the `3dsecure` sub-dictionary.
```
Comment (mt-clearhaus) [inline website/content/gateway/api_reference/resources/authorizations/authorizations_methods/googlepay.md:36]:
```suggestion
**Notice**: An authorization made with `googlepay` is strongly authenticated (SCA in PSD2) if `authMethod` is `CRYPTOGRAM_3DS` and the [Google Pay guidelines for SCA](https://developers.google.com/pay/api/android/guides/resources/sca) has been followed. If `authMethod` is `PAN_ONLY`, a 3-D Secure flow is required for SCA and the resulting ARes/RReq must be supplied in the `3dsecure` sub-dictionary.
```

File: website/content/gateway/api_reference/resources/authorizations/authorizations_methods/googlepay.md [hunk 2]
```diff
@@ -24,14 +24,21 @@ The shared secret derived from the ephemeral public key and your private key
...
+
+**Notice**: The `3dsecure` sub-dictionary is not supported if `authMethod` is `CRYPTOGRAM_3DS`.
 
 **Notice**: An authorization made with `googlepay` cannot be a subsequent-in-series authorization.
 
-**Notice**: The recipient_id for the `googlepay` test environment is `merchant:12345678901234567890`. 
+**Notice**: The `recipient_id` for the `googlepay` test environment is `merchant:12345678901234567890`.
```
Comment (mt-clearhaus) [inline website/content/gateway/api_reference/resources/authorizations/authorizations_methods/googlepay.md:42]:
```suggestion
**Notice**: The `recipient_id` for `googlepay` in the test environment is `merchant:12345678901234567890`.
```

File: website/content/gateway/api_reference/resources/authorizations/authorizations_methods/googlepay.md (hunk 1 above)
Comment (mt-clearhaus) [reply website/content/gateway/api_reference/resources/authorizations/authorizations_methods/googlepay.md:36]: I retract this suggestion :slightly_smiling_face: 

Screenshot from 2023-12-13 14-42-52

Screenshot from 2023-12-13 14-43-13

Screenshot from 2023-12-13 14-42-57

Screenshot from 2023-12-13 14-43-19

**Review by mt-clearhaus — APPROVED** [review]

**Pull request merged**